.github/workflows/operator-publish-operator-hub.yml

```diff
@@ -0,0 +1,23 @@
+name: "Publish release on operator hub"
+on:
+  release:
+    types: [published]
+
+jobs:
+  operator-hub-prod-release:
+    if: startsWith(github.event.release.tag_name, 'operator/')
+    uses: ./.github/workflows/operator-reusable-hub-release.yaml
+    with:
+      org: redhat-openshift-ecosystem
+      repo: community-operators-prod
+    secrets:
+      GRAFANABOT_GITHUB_TOKEN: ${{ secrets.GRAFANABOT_GITHUB_TOKEN }}
```

Comment:
I don't think we have `GRAFANABOT_GITHUB_TOKEN`, but `GH_TOKEN` should work (that a grafanabot PAT we're using for releases)